Angular2WebpackVisualStudio/src/Angular2WebpackVisualStudio/webpack.prod.js

```diff
@@ -84,18 +85,17 @@ module.exports = {
         }),
         new webpack.optimize.CommonsChunkPlugin(
             {

```

Comment:
Due to the fact that this is a demo application for me having one request more/less does not matter. For me its more interesting either clearify _why_ we got rid of the chunks in the first place or we put them in again. Because this example should be transparent, easy to follow and not cause questions :)